mne/viz/backends/_pyvista.py

```diff
@@ -339,7 +341,7 @@ def surface(self, surface, color=None, opacity=1.0,
                 opacity=opacity,
                 cmap=cmap,
                 backface_culling=backface_culling,
-                smooth_shading=self.figure.smooth_shading
+                smooth_shading=self.figure.smooth_shading,
```

Comment:
Shouldn't this just be `False` since they're either added or computed above?

Reply:
I remove it in the commit just after